[338] Previewing examples with contextual HTML (#353)
github/view_component: CHANGELOG.md

# master

> * Add support for templates as preview examples

Reviewer: ```suggestion
* Add support for templates as ViewComponent::Preview examples.
```

> * Add support for ERB templates as preview examples.

Reviewer: Do we want to support all template handlers, not just ERB?

Author: Yeah, given we are just calling render in the controller I think we are good to say that we support any handle, no?

Reviewer: Might we update this message to reflect that all handlers are supported?

> * Add support to use erb templates as preview examples

Reviewer: ```suggestion
* Add support for ERB templates as preview examples.
```
